### DISCUSSION TITLE
Friendly name on groups

### DISCUSSION THREAD

**Discussion by ghost** [LironEr/bundlemon#101] [Ideas] [answered]
Groups can sometimes have very complex glob patterns, which might make it harder for someone not familiar with the code base to correctly identify what it does/ is responsible for.
I had a case where I was grouping all my app JS files, with the exception of translations, third-party code and the main `app.js` file. The glob works as expected, but the generated reports make it difficult to understand.
My proposal is to extend the [File](https://github.com/LironEr/bundlemon/blob/master/docs/types.md#File) type to also support a "friendy name" kind of property where we can name each group. That could also apply to single files too.

**Comment by LironEr** [comment]
Sounds like a really good idea, hopefully I will be able to implement it next weekend 😊

**Comment by LironEr** [comment, marked as answer]
This feature was just released in [bundlemon v1.4.0](https://github.com/LironEr/bundlemon/releases/tag/bundlemon%401.4.0)

Thank you for your suggestion (:
